tools/install.sh

```diff
@@ -38,16 +39,17 @@ while [ "$1" != "" ]; do
             exit 1
             ;;
     esac
+    INERACTIVE=false
     shift
 done
 
-if [ ! $domain ]; then
+if [ ! $domain ] && [ $INERACTIVE = true ]; then
     read -p "Enter exoframe-server domain: " domain
 fi
-if [ ! -f "$FILE" ] && [ $ssl = false ]; then
+if [ ! -f "$FILE" ] && [ $ssl = false ] && [ $INERACTIVE = true ]; then
     read -p "Enter email to enable SSL support: " ssl
 fi
-if [ ! $passvar ]; then
+if [ ! $passvar ] && [ $INERACTIVE = true ]; then
```

Comment:
I think script should fail / throw error if it's in non-interactive mode and some params are missing (e.g. pass, domain)

Reply:
Currently if some params are missing it asks for input :/

Reply:
I'd use the following logic:
- if there are _any_ flags - consider it non-interactive and fail if not enough args are passed
- if there are no flags - we're running interactive, so ask questions